<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FreqDoubWgtClass</class>
 <widget class="QWidget" name="FreqDoubWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>903</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FreqDoubWgt</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="mainTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>倍频相关性分析</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>相关频率分布</string>
           </property>
           <layout class="QHBoxLayout" name="HLayout_Cht_FreqDoubDistribute"/>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>倍频尺寸统计</string>
            </attribute>
            <layout class="QHBoxLayout" name="HLayout_Cht_FreqDoubCount"/>
           </widget>
           <widget class="QWidget" name="tab_4">
            <attribute name="title">
             <string>父级倍频分布</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widget_panel" native="true">
         <property name="maximumSize">
          <size>
           <width>260</width>
           <height>16777215</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>FFT参数</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_SampleRate">
                 <property name="text">
                  <string>采样率</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="SBox_SampleRate">
                 <property name="minimumSize">
                  <size>
                   <width>72</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>72</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="maximum">
                  <number>96000</number>
                 </property>
                 <property name="value">
                  <number>44100</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_WndPtCount">
                 <property name="text">
                  <string>窗口点数</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="SBox_WndPtCount">
                 <property name="minimumSize">
                  <size>
                   <width>72</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>72</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="maximum">
                  <number>9192</number>
                 </property>
                 <property name="value">
                  <number>4096</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_MaxEffectFreq">
                 <property name="text">
                  <string>最大接受频率</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="SBox_MaxEffectFreq">
                 <property name="minimumSize">
                  <size>
                   <width>72</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>72</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="maximum">
                  <number>22100</number>
                 </property>
                 <property name="value">
                  <number>20000</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_WndSpecConstrainFreqDriftRatio">
                 <property name="text">
                  <string>窗谱抑制峰值偏移（%）</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="DSBox_WndSpecConstrainFreqDriftRatio">
                 <property name="minimumSize">
                  <size>
                   <width>72</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>72</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="maximum">
                  <double>50.000000000000000</double>
                 </property>
                 <property name="value">
                  <double>25.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="tabBarAutoHide">
             <bool>false</bool>
            </property>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>倍频张量参数</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_TensorEndFreq">
                  <property name="text">
                   <string>终止频率(hz)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="DSBox_TensorEndFreq">
                  <property name="minimumSize">
                   <size>
                    <width>72</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>72</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="maximum">
                   <double>20000.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>4186.009000000000015</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_ParentMatrixIncludeMainFreqCount">
                  <property name="text">
                   <string>父级包含前N个频率</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="SBox_ParentMatrixIncludeMainFreqCount">
                  <property name="minimumSize">
                   <size>
                    <width>72</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>72</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="value">
                   <number>3</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_MaxMultiTime">
                  <property name="text">
                   <string>最大倍频次数</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="SBox_MaxMultiTime">
                  <property name="minimumSize">
                   <size>
                    <width>72</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>72</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="value">
                   <number>50</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_TensorStartFreq">
                  <property name="text">
                   <string>起始频率(hz)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="DSBox_TensorStartFreq">
                  <property name="minimumSize">
                   <size>
                    <width>72</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>72</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="decimals">
                   <number>3</number>
                  </property>
                  <property name="maximum">
                   <double>20000.000000000000000</double>
                  </property>
                  <property name="value">
                   <double>65.406000000000006</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_cutOffOnOverLappingStartPos">
                <property name="text">
                 <string>是否去除自交叠倍频区域</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>张量状态</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLabel" name="label_TensorResult_StartFreq">
               <property name="text">
                <string>张量起始频率：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_TensorResult_EndFreq">
               <property name="text">
                <string>张量终止频率：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_TensorResult_MatrixCount">
               <property name="text">
                <string>倍频矩阵个数：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_TensorResult_MatrixFreqInterval">
               <property name="text">
                <string>矩阵频率间隔：</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_UpdateChtAfterGetTensor">
            <property name="text">
             <string>同时刷新图表</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_tensorGenState">
            <property name="text">
             <string>分析状态</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar_TensorGenState">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_GenTensor">
            <property name="text">
             <string>生成张量</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_Cht_ShowMatricFreq">
              <property name="text">
               <string>显示中心频率</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="SBox_Cht_ShowMatricFreq">
              <property name="minimumSize">
               <size>
                <width>72</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>72</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>20000</number>
              </property>
              <property name="value">
               <number>500</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_Cht_ShowMultipleTime">
              <property name="text">
               <string>显示倍频次数</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="SBox_Cht_ShowMultipleTime">
              <property name="minimumSize">
               <size>
                <width>72</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>72</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="value">
               <number>20</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_cht_IsMatrixInsertFreq">
            <property name="text">
             <string>这是不是一个插入频率点：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Cht_ShowFreqPtIndex">
            <property name="text">
             <string>分析频率FFT点索引：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Cht_ShowFreq">
            <property name="text">
             <string>分析频率FFT点频率：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Cht_ShowMatricMaxMultTime">
            <property name="text">
             <string>有效倍频次数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Cht_selfOverlappintStartMultiTime">
            <property name="text">
             <string>首个自交叠区域倍频次数：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_updateTensorCht">
            <property name="text">
             <string>刷新图表</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>网络输出</string>
      </attribute>
      <layout class="QVBoxLayout" name="VLayout_ChtPanel"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
